apps/src/templates/studioHomepages/CoteacherInviteNotification.jsx

```diff
@@ -18,15 +18,34 @@ export const showCoteacherInviteNotification = coteacherInvite => {
   return !!coteacherInvite && DCDO.get('show-coteacher-ui', true);
 };
 
+export const showCoteacherForPlInviteNotification = coteacherInviteForPl => {
+  return !!coteacherInviteForPl && DCDO.get('show-coteacher-ui', true);
+};
+
 const CoteacherInviteNotification = ({
+  isForPl,
   asyncLoadCoteacherInvite,
   asyncLoadSectionData,
   coteacherInvite,
+  coteacherInviteForPl,
 }) => {
-  if (!showCoteacherInviteNotification(coteacherInvite)) {
+  if (
+    !(
+      showCoteacherInviteNotification(coteacherInvite) ||
+      showCoteacherForPlInviteNotification(coteacherInviteForPl)
+    )
+  ) {
     return null;
   }
 
+  let invite;
```

Comment:
Can we wrap this in a react function (`React.useMemo` or `React.useCallback`) to prevent it from re-rendering everything on every render